PostHog/PostHogSDK.swift

```diff
@@ -887,10 +887,26 @@ let maxRetryDelay = 30.0
         }
 
         if shouldCapture {
-            let properties = [
+            let requestId = featureFlags?.lastRequestId ?? ""
+            let details = featureFlags?.getFeatureFlagDetails(flagKey)
+
+            var properties = [
                 "$feature_flag": flagKey,
                 "$feature_flag_response": flagValue ?? "",
+                "$feature_flag_request_id": requestId,
```

Comment:
we do set `"$feature_flag_response"` if the value is nil with an empty string.
now we do the same with `$feature_flag_request_id`, maybe we should just not send those values

Reply:
> we do set "$feature_flag_response" if the value is nil with an empty string.

Oh, that's actually incorrect. We want to know if the value is `nil` because that means something. I'll fix that.